### ISSUE TITLE
Think of blockchain as a relatively slow, very expensive database that provides excellent resistance to hacking and corruption. It's a Write-Once, Read Mostly (WORM) system.

### ISSUE THREAD

**Issue by weberius** [weberius/distributedLedgerTechnologies#1]
You absolutely could store any data you want in a hypothetical blockchain. The practical limits are, you don't want to store very large chunks of data (so, not video); you probably don't want to store frequently changing data (so, not a thesis paper you're revising) -- unless it's important somehow to record every single change forever.

Because, its other feature is, once something is written to a blockchain, it's there forever.

Need to fix a typo? Then you add a new record with a correction.

Need to delete a record? Too bad, you can't. Best you can do is enter a new record saying that the record you wish to delete is "obsolete" or "repudiated" or "no longer valid" or "should be considered as deleted."

In short, it's wise to treat your blockchain as a permanent record.

Is Blockchain a distributed database? [closed]
